README.md

```diff
@@ -49,8 +49,8 @@ PyCM is the swiss-army knife of confusion matrices, targeted mainly at data scie
 	</tr>
     <tr>
 		<td align="center">CI</td>
-		<td align="center"><img src="https://github.com/sepandhaghighi/pycm/workflows/CI/badge.svg?branch=master"></td>
-		<td align="center"><img src="https://github.com/sepandhaghighi/pycm/workflows/CI/badge.svg?branch=dev"></td>
+		<td align="center"><img src="https://github.com/sepandhaghighi/pycm/actions/workflows/test.yml/badge.svg?branch=master"></td>
```

Comment:
This one is not working, right?

Reply:
This will be resolved after updating the `master` branch.